devops/ansible/examples/girder-dev-environment/site.yml

```diff
@@ -70,3 +70,11 @@
       become: yes
       become_user: root
       when: ansible_os_family == "RedHat"
+
+    - name: Activate virtualenv on login
+      lineinfile:
+        dest: ~/.profile
```

Comment:
We should not use `~` here, since that's a Bash construct. Ansible provides the fact `ansible_user_dir`, which represents the location more robustly.